request/middleware.py

```diff
@@ -6,6 +8,8 @@
 from .router import Patterns
 from .utils import request_is_ajax
 
+logger = logging.getLogger('django.security.request')
```

Comment:
Is there a particular reason you chose this logging namespace?

My issue with this is that it isn't a logging namespace that you control, so Django make someday need to use this namespace, which would break this logger. For people who want to apply different logging settings for this package and module, it's also not immediately clear where this logging message is coming from, and the warning message doesn't really help with that.

I would prefer to just stick with [the standard](https://docs.djangoproject.com/en/4.0/howto/logging/#make-a-basic-logging-call):

```suggestion
logger = logging.getLogger(__name__)
```

That makes it immediately obvious where the logged warning is being emitted from, and that effective namespace (`django_request.middleware` IIRC) _is_ something that we will presumably retain control of, so there's less chance of it breaking in the future.

Reply:
I would prefer to use a custom namespace with a `security` in it. Changed to `request.middleware.security`.

Reply:
That's an acceptable compromise to me. Thank you!